### ISSUE TITLE
scalar bar as separate image, or outside window

### ISSUE THREAD

**Issue by dansand** [pyvista/pyvista-support#323]
It would be great to have the capability to either save the scalar bar as a separate figure, or to place the scalar bar outside the window. 

For example, it would be useful to be able to emulate the figure/colorbar arrangements discussed here:

https://stackoverflow.com/questions/13784201/matplotlib-2-subplots-1-colorbar

Thanks

**Comment by banesullivan** [comment]
I agree this would be nice. I will move this to PyVista as a feature request.

I think we could find a way to render the scalar bar by itself and save it as a separate image on demand.
